apps/system/js/nfc_handover_manager.js

```diff
@@ -238,7 +238,12 @@
           this.settingsNotified = true;
         }
       } else {
-        action.callback.apply(this, action.args);
+        Service.request('Bluetooth:adapter').then((adapter) => {
+          this._adapter = adapter;
+          action.callback.apply(this, action.args);
+        }).catch((err) => {
```

Comment:
though its not called now, add the catch in case one day bluetooth provide api to reject the adapter request.